docs/user/nifi/examples.rst

```diff
@@ -0,0 +1,55 @@
+Examples
+--------
+
+Ingest a GeoMesa Kafka Topic into HBase
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A common use case is to store the most recent state of an object in a GeoMesa Kafka data store to drive a live map
+view, but to persist every state of the object into long-term storage in HBase or Accumulo for history and
```

Comment:
I think the "but" could be an "and" here.

Reply:
```suggestion
view, and to persist every state of the object into long-term storage in HBase or Accumulo for history and
```